content/blog/2024/dei-leadership-strategy/index.md

```diff
@@ -26,11 +26,11 @@ We've also invested in creating an inclusive organizational culture via document
 
 ## The importance of diverse _leadership_
 
-However, over the past few months we have reflected on our organization's structure and future plans, and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
+However, over the past few months, we have reflected on our organization's structure and future plans and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
```

Comment:
@choldgraf very minor point -- "noticed" implies it's suddenly occurred to us that there are very few women in the organisation. I don't feel confident that this accurately describes our position and how we got here. Rather, my assumption is that Jenny's original wording here better speaks to the fact that it's clearly an issue, but we have yet to solve it. This post, to my eyes, is saying "we need to solve it now". 

Please, _don't let me speak over more qualified voices on this subject_ (I have probably not been in any of these conversations), and _reject this if you think I've missed the point_, but I'll suggest a reword?

```suggestion
However, over the past few months, we have reflected on our organization's structure and future plans. In particular, we recognize that there is a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
```

Reply:
That works for me - this is something I've been stressed out about for a long time. I think that the slide deck sharpened attention on this point, as opposed to made it understood for the first time.